ci-operator/step-registry/cucushift/installer/rehearse/aws/usgov/ipi/private/ep/cucushift-installer-rehearse-aws-usgov-ipi-private-ep-workflow.metadata.json

```
@@ -0,0 +1,10 @@
+{
+	"path": "cucushift/installer/rehearse/aws/usgov/ipi/private/ep/cucushift-installer-rehearse-aws-usgov-ipi-private-ep-workflow.yaml",
```

```suggestion
	"path": "cucushift/installer/rehearse/aws/usgov/ipi/private/with-ep-s3/cucushift-installer-rehearse-aws-usgov-ipi-private-with-ep-s3-workflow.yaml",
```
To keep consistent with `aws/ipi/disconnected/private/cco-manual-security-token-service/private-s3/with-ep-sts-ec2-elb`

---

Updated.